app/javascript/utilities/http/request.js

```diff
@@ -25,7 +27,7 @@ export async function request(url, options = {}) {
     headers,
     body,
     method = 'GET',
-    csrfToken = await getCsrfToken(),
+    csrfToken = await getCSRFToken(),
```

Comment:
We're going to use the imported pack module instead of the global method, this will allow us to now use methods from this `request` file in the admin layouts